### ISSUE TITLE
Update AWS "composite" "managers" to convention

### ISSUE THREAD

**Issue by bennett000** [rangle/the-clusternator#334]
This concerns prManager, projectManager (our external API) and deploymentManager

This is similar in some ways to #333 but with different files.  There's more room for improvement here though since these files were supposed to be temporary in the first place!

Ideally this should be done after #333 so that changes can best take advantage of everything returning functions.

One thing that could be improved significantly is cleaning up on failures.  That's another ticket and it's out there
